tilequeue/command.py

```diff
@@ -1075,6 +1072,26 @@ def delete_tile_of_interest(s3_parts, coord_ints):
     logger.info('Removing %s tiles from TOI and S3 ... done',
                 len(toi_to_remove))
 
+    logger.info('Computing tiles to add ...')
+    toi_to_add = new_toi - tiles_of_interest
+    logger.info('Computing tiles to add ... done. %s found',
+                len(toi_to_add))
+
+    logger.info('Adding %s tiles to TOI and SQS ...',
+                len(toi_to_add))
+
+    def add_tile_of_interest(coord_int):
+        # Add to Redis
+
+        # Enqueue a render request for the tile
+        pass
+
+    for coord_int in toi_to_add:
+        add_tile_of_interest(coord_int)
```

Comment:
Ok, you can probably look at the [enqueue tiles of interest command](https://github.com/tilezen/tilequeue/blob/9fa942f41adad206fabe473a693207a9548eb17d/tilequeue/command.py#L932-L952) for inspiration.